applications/luci-app-omcproxy/root/etc/uci-defaults/40_omcproxy

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# needed for "Save and Apply" to restart iptvhelper
```

Comment:
This file should not be needed at all since `omcproxy` is already [procd enabled](https://github.com/openwrt/openwrt/blob/master/package/network/services/omcproxy/files/omcproxy.init#L128).